### PR TITLE
Release 1.0.0

### PR DESCRIPTION
**Pull request by mpgxvii**
- Add Grafana provisioning files for data sources and dashboards so that these are automatically added on startup
- Add Docker files for Grafana, JDBC Connector, TimescaleDB, and Kafka dependencies

### CODE REVIEW

File: docker/kafka-wait [hunk 1]
```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```
Comment (blootsvoets) [inline docker/kafka-wait]: This file is not used?

File: docker/launch [hunk 2]
```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
```
Comment (blootsvoets) [inline docker/launch]: This file is not used?

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+    environment:
+      GF_SECURITY_ADMIN_PASSWORD: ${GRAFANA_PASSWORD}
+      GF_SERVER_ROOT_URL: ${GRAFANA_URL}
+      GF_USERS_VIEWERS_CAN_EDIT: 'true'
+      GF_INSTALL_PLUGINS: 'flant-statusmap-panel'
+      GF_USERS_DEFAULT_THEME: light
+      GF_DASHBOARDS_DEFAULT_HOME_DASHBOARD_PATH: '/etc/grafana/provisioning/dashboards/home.json'
+      POSTGRES_DB: ${POSTGRES_DB}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+      POSTGRES_HOST: host.docker.internal:5433
```
Comment (blootsvoets) [inline docker-compose.yml]: Not sure about this one... Should be `timescaledb:5432`?

**Review by blootsvoets — APPROVED** [review]
LGTM in principle, not sure about some variables, but if they're tested it should be OK

File: docker/kafka-wait (hunk 1 above)
Comment (mpgxvii) [reply docker/kafka-wait]: Made the changes.

File: docker/launch (hunk 2 above)
Comment (mpgxvii) [reply docker/launch]: Made the changes.

File: docker-compose.yml (hunk 3 above)
Comment (mpgxvii) [reply docker-compose.yml]: Made the changes.

**Pull request merged**